loama/src/components/LoButton.vue

```diff
@@ -48,16 +50,30 @@ button {
     border-radius: var(--base-corner);
     height: fit-content;
     border: none;
+    cursor: pointer;
+    border: .2em solid;
+}
+
+button[disabled] {
+    background-color: grey;
+    cursor: not-allowed;
+}
```

Comment:
You can't really trigger this through the components *(Atleast, I don't see an prop for triggering the disable state)*
You can leave it, but keep it in mind.

Also, we have a grey specified in the `css` variables
```suggestion
button[disabled] {
    background-color: var(--lama-gray);
    cursor: not-allowed;
}
```